pkg/ext-proc/metrics/README.md

```diff
@@ -6,13 +6,40 @@ This documentation is the current state of exposed metrics.
 * [Exposed Metrics](#exposed-metrics)
 * [Scrape Metrics](#scrape-metrics)
 
+## Requirements
+
+In order to process response metrics, you will need to enable `Buffered` mode for response in `EnvoyExtensionPolicy`:
```

Comment:
Pls mention that there is a work to enable both streaming and buffered mode https://github.com/kubernetes-sigs/gateway-api-inference-extension/issues/178

Reply:
updated